python_modules/dagster/dagster/_config/pythonic_config/__init__.py

```diff
@@ -162,7 +162,34 @@ def _is_field_internal(self, name: str) -> bool:
 
 
 class Config(MakeConfigCacheable):
-    """Base class for Dagster configuration models."""
+    """Base class for Dagster configuration models, used to specify config schema for
+    ops and assets. Subclasses `pydantic.BaseModel`.
```

Comment:
Is it possible to format `pydantic.BaseModel` as code instead of italics?

Reply:
fixed!